power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/vnf_pgm_converter.h

```diff
@@ -11,10 +11,31 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+
+/**
+ * @brief Create the PGM_VNF_converter
+ * @param handle
+ * @param file_buffer A pointer to the file buffer object which holds the .vnf data.
+ * @return The pointer to a PGM_VNF_converter object.
```

Comment:
Nitpick: I would change object for instance here. Please check other `@return` in the other docstrings as well.

Reply:
Both read perfectly fine to me.

Reply:
That is why is a nitpick. Probably style. Not needed to update if don't think it doesn't add anything.